lib/migration.js

```diff
@@ -437,28 +364,9 @@ function mixinMigration(Oracle) {
     async.series(tasks, cb);
   };
 
-  /*!
-   * Find the column type for a given model property
-   *
-   * @param {String} model The model name
-   * @param {String} property The property name
-   * @returns {String} The column type
-   */
-  Oracle.prototype.columnDataType = function(model, property) {
-    var columnMetadata = this.columnMetadata(model, property);
-    var colType = columnMetadata && columnMetadata.dataType;
-    if (colType) {
-      colType = colType.toUpperCase();
-    }
-    var prop = this.getModelDefinition(model).properties[property];
-    if (!prop) {
-      return null;
-    }
-    var colLength = columnMetadata && columnMetadata.dataLength || prop.length;
```

Comment:
in base connector `colLength` can also read from `prop.limit`
```js
var colLength = columnMetadata && columnMetadata.dataLength ||
 prop.length || prop.limit;
```
IIRC this is a mysql only implementation, I am not aware of any problem  caused by applying it to all connectors, so it looks good to me. Appreciate suggestions/opinions from others :)